pymc/tests/helpers.py

```diff
@@ -148,3 +155,66 @@ def assert_random_state_equal(state1, state2):
         (in2out(local_check_parameter_to_ninf_switch), -1)
     )
 )
+
+
+class TestStepMethod:
```

Comment:
I think pytest will run this baseclass when it finds it because it starts with `Test`